test/widgets/test_volume.py

```diff
@@ -51,14 +51,21 @@ def test_emoji():
     vol._update_drawer()
     assert vol.text == "\U0001f50a"
 
+    vol.mute = "[off]"
+    vol._update_drawer()
+    assert vol.text == "\U0001f507"
+
 
 def test_text():
     fmt = "Volume: {}"
     vol = Volume(fmt=fmt)
-    vol.volume = -1
+    vol.volume = 80
     vol._update_drawer()
-    assert vol.text == "M"
+    assert vol.text == "80%"
```

Comment:
I'd leave this as `-1` and correct the output for that scenario. `-1` now represents an error so maybe we should tell the user.

Reply:
Actually, as per comment above, `-1` also means muted for the Pulse volume widget so we should continue to treat that way.